pages/Nix/Hyprland on Home Manager.md

```diff
@@ -137,13 +137,10 @@ module, or the flake-based Home Manager module.
         # binds $mod + [shift +] {1..10} to [move to] workspace {1..10}
         builtins.concatLists (builtins.genList (
             x: let
-              ws = let
-                c = (x + 1) / 10;
-              in
-                builtins.toString (x + 1 - (c * 10));
+              ws = builtins.toString (x + 10);
```

Comment:
Based on the [keycodes](https://github.com/xkbcommon/libxkbcommon/blob/master/include/xkbcommon/xkbcommon-keysyms.h#L645) shouldn't this be `+ 48`?